builder/static_code.go

```diff
@@ -869,7 +869,7 @@ func listJoin(list []string, sep string, lastSep string) string {
 	case 1:
 		return list[0]
 	default:
-		return fmt.Sprintf("%s %s %s", strings.Join(list[:len(list)-1], sep), lastSep, list[len(list)-1])
+		return strings.Join(list[:len(list)-1], sep) + " " + lastSep + " " + list[len(list)-1]
```

Comment:
If we care about performance, maybe we should also handle the case `len(list) == 2` in the `switch`.
This would be something like:

```go
case 2:
    return list[0] + " " + lastSep + " " + list[1]
```